semgrep/semgrep/config_resolver.py

```diff
@@ -144,25 +139,29 @@ def _safe_relative_to(a: Path, b: Path) -> Path:
 
     @staticmethod
     def _convert_config_id_to_prefix(config_id: str) -> str:
+        if ADJUST_FOR_DOCKER:
```

Comment:
Can you add a comment that this is so that the rule-ids are not prepended with src (and also  that this assumes the rules will be somewhere in SEMGREP_SRC_DIRECTORY)